src/Onboarding/Setup/templates/index.html.php

```diff
@@ -79,6 +79,19 @@
 								)
 							),
 						]
+					)
+					: $this->render_template(
+						'row-item',
+						[
+							'class'       => 'stripe stripe-webhooks',
+							'icon'        => $this->image( 'stripe-connect@2x.min.png' ),
+							'icon_alt'    => esc_html__( 'Stripe', 'give' ),
+							'title'       => esc_html__( 'Please configure your Stripe webhook to finalize the setup.', 'give' ),
+							'description' => esc_html__( 'In order for Stripe to function properly, you must add a new Stripe webhook endpoint. To do this please visit the Webhooks Section of your Stripe Dashboard and click the Add endpoint button and paste the following URL: ', 'give' ) . '<br /><span id="stripeWebhooksCopyHandler" class="stripe-webhooks-url"><input disabled="disabled" id="stripeWebhooksCopy" value="https://wordpress.test?give-listener=stripe" /> &nbsp; <i id="stripeWebhooksCopyIcon" class="fa fa-clipboard"></i></input>',
```

Comment:
Looks like you are using a hardcoded site URL of `wordpress.test`, I figure you just forgot to swap this out for the actual site URL.

Reply:
Resolved by 9f01392